### PR TITLE
Added doc blocks in each file and formatted

### PR DESCRIPTION
**Pull request by senaria**

### CODE REVIEW

File: Importer/Configuration/RelationshipConfiguration.php [hunk 1]
```diff
@@ -2,57 +2,86 @@
...
     public function getOwnerLookupConfigurationField()
     {
         return $this->ownerLookupConfigurationField;
     }
 
     /**
-     * @param mixed $ownerLookupConfigurationField
+     * @param LookupFieldConfiguration $ownerLookupConfigurationField
      */
     public function setOwnerLookupConfigurationField($ownerLookupConfigurationField)
```
Comment (senaria) [inline Importer/Configuration/RelationshipConfiguration.php]: add type

File: Importer/Interpreter/Error/Handler/FileLoggerErrorHandler.php [hunk 2]
```diff
@@ -70,18 +87,37 @@ public function handle(InterpreterException $exception, $index, $rowData)
...
+    {
+        $indentation = str_repeat("\t", $indentation);
+        fwrite($this->output, $indentation . $message . PHP_EOL);
+    }
+
+    /**
+     * @param ExecutionContextInterface $exception
+     * @return mixed
+     */
+    protected function buildInvalidEntityViolationsMessages(ExecutionContextInterface $exception)
```
Comment (senaria) [inline Importer/Interpreter/Error/Handler/FileLoggerErrorHandler.php]: check

File: Importer/Interpreter/Exception/UnknownColumnException.php [hunk 3]
```diff
@@ -4,6 +4,9 @@
 
 class UnknownColumnException extends InterpreterException
 {
+    /**
+     * @var mixed
```
Comment (senaria) [inline Importer/Interpreter/Exception/UnknownColumnException.php]: check

File: Importer/Interpreter/InterpreterInterface.php [hunk 4]
```diff
@@ -6,7 +6,21 @@
 
 interface InterpreterInterface
 {
-    public function interpret($data, $associative = true);
+    /**
+     * @param array $data
+     * @param bool  $associative
+     * 
+     * @return object[]|void
```
Comment (senaria) [inline Importer/Interpreter/InterpreterInterface.php]: null or object

File: Command/GenerateImportConfigCommand.php [hunk 5]
```diff
@@ -21,6 +21,13 @@ protected function configure()
             ->addOption('entity', 'en', InputOption::VALUE_REQUIRED, 'Enter the name of the entity');
     }
 
+    /**
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     * 
+     * @return int
```
Comment (mkruk-u2) [inline Command/GenerateImportConfigCommand.php]: Or nothing.
Btw, weird return value.

File: DependencyInjection/CompilerPass/ImportEventSubscribersCompilerPass.php [hunk 6]
```diff
@@ -9,10 +9,8 @@
 class ImportEventSubscribersCompilerPass implements CompilerPassInterface
 {
     /**
-     * You can modify the container here before it is dumped to PHP code.
-     *
      * @param ContainerBuilder $container
-     *
+     *
```
Comment (mkruk-u2) [inline DependencyInjection/CompilerPass/ImportEventSubscribersCompilerPass.php]: Is there a space here?

File: Importer/CsvImporter.php [hunk 7]
```diff
@@ -7,26 +7,53 @@
...
         parent::__construct($configuration, $interpreter, $entityManager, $eventDispatcher);
         $this->delimiter = $delimiter;
     }
 
+    /**
+     * @param string $filename
+     * @param bool   $hasHeaders
+     * @param bool   $flush
+     *
+     * @return array|null|\object[]
```
Comment (mkruk-u2) [inline Importer/CsvImporter.php]: Here

File: Importer/Error/CsvHeadersError.php [hunk 8]
```diff
@@ -4,18 +4,30 @@
...
 class CsvHeadersError implements ImporterErrorInterface
 {
+    /**
+     * @var array
+     */
     private $invalidHeaders = [];
 
-    public function __construct($invalidHeaders)
+    /**
+     * @param $invalidHeaders
```
Comment (mkruk-u2) [inline Importer/Error/CsvHeadersError.php]: array type missing

File: Importer/Interpreter/Field/LookupFieldInterpreter.php [hunk 9]
```diff
@@ -105,9 +126,8 @@ private function internalLookup($class, $lookupField, $value)
 
     /**
      * @param FieldConfigurationInterface $fieldConfiguration
-     * @param                             $value
-     * @param                             $exception
-     *
+     * @param mixed                            $value
+     * @param \Exception                            $exception
```
Comment (mkruk-u2) [inline Importer/Interpreter/Field/LookupFieldInterpreter.php]: Indenting is messed up a bit here.

**Pull request merged**